core/capabilities/testutils/backend.go

```diff
@@ -0,0 +1,118 @@
+package testutils
+
+import (
+	"context"
+	"encoding/json"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind/backends"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/eth/ethconfig"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/assets"
+	evmclient "github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/headtracker"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/logpoller"
```

Comment:
If our shared `testutils` package is importing `evm`, doesn't that break our chain agnostic separation? I think geth alone could be problematic, but IIUC importing an actual chain relayer implementation simply won't be possible in the future.

Reply:
Yes, that is why this capability is part of chainlink core repo, instead of, in the new capabilities repo. EVM relayer has not yet been LOOPified.
To be able to test with EVM relayer and gain confidence of the log event trigger implementation.

Reply:
This helper file (and likely the tests using it) should be in an EVM package, not a common package.

Reply:
Moved this test to ./core/servicesrelay/evm as discussed